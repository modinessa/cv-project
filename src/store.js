import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
