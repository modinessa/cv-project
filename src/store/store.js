import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});