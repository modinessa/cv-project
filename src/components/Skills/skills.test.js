import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Skills } from './index';


describe('Skills', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Skills />
			</Provider>
		)).toMatchSnapshot();
	})
})