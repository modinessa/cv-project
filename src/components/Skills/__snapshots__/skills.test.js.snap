// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills renders component 1`] = `
<section
  className="app-section"
  id="skills"
>
  <div
    className="skills-title"
  >
    <h2
      className="app-title"
    >
      Skills
    </h2>
    <button
      className="app-button"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-pencil "
        data-icon="pencil"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M421.7 220.3L188.5 453.4L154.6 419.5L158.1 416H112C103.2 416 96 408.8 96 400V353.9L92.51 357.4C87.78 362.2 84.31 368 82.42 374.4L59.44 452.6L137.6 429.6C143.1 427.7 149.8 424.2 154.6 419.5L188.5 453.4C178.1 463.8 165.2 471.5 151.1 475.6L30.77 511C22.35 513.5 13.24 511.2 7.03 504.1C.8198 498.8-1.502 489.7 .976 481.2L36.37 360.9C40.53 346.8 48.16 333.9 58.57 323.5L291.7 90.34L421.7 220.3zM492.7 58.75C517.7 83.74 517.7 124.3 492.7 149.3L444.3 197.7L314.3 67.72L362.7 19.32C387.7-5.678 428.3-5.678 453.3 19.32L492.7 58.75z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
       
      <span
        className="text"
      >
        Open edit
      </span>
    </button>
  </div>
  <div
    className="skill-ranges"
  >
    <div>
      <div
        className="ruler-container"
      >
        <div
          className="mark-25"
        />
        <div
          className="mark-50"
        />
        <div
          className="mark-75"
        />
      </div>
      <div
        className="ruler-labels"
      >
        <div
          className="item-1"
        >
          Beginner
        </div>
        <div
          className="item-2"
        >
          Proficient
        </div>
        <div
          className="item-3"
        >
          Expert
        </div>
        <div
          className="item-4"
        >
          Master
        </div>
      </div>
    </div>
  </div>
</section>
`;
